docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -0,0 +1,163 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+## Installing SpECTRE on Apple Silicon {#installation_on_apple_silicon}
+
+The following instructions show how to install SpECTRE and its dependencies
+on Apple Silicon Macs. Apple Silicon is an arm64 architecture, in contrast
+to the x86-64 architecture that SpECTRE usually targets. These instructions
+will result in an Apple Silicon native build of SpECTRE.
+
+\note Floating-point exception trapping and python bindings are not currently
+supported on Apple Silicon.
+
+### 0. Install the xcode command-line tools.
+
+Install the xcode command-line tools, which include the clang compiler, etc.
+
+~~~
+xcode-select --install
+~~~
+
+### 1. Make a directory to install prerequisites
+
+First, make a directory to hold some prerequisites that spectre depends on.
+Name this directory whatever you like, e.g. `$HOME/apps`.
+~~~
+export SPECTRE_DEPS_ROOT=$HOME/apps
+mkdir $SPECTRE_DEPS_ROOT
+cd $SPECTRE_DEPS_ROOT
+mkdir src
+cd src
+~~~
+
+### 2. Install python dependencies
+
+Spectre depends on python and some python packages. Miniforge is a way to
+install an arm64-native python stack on Apple Silicon Macs. Download
+the [Miniforge3-MacOSX-arm64.sh](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh)
+installation script, and then do the following:
+~~~
+# Install miniforge. Accept all default choices
+bash Miniforge3-MacOSX-arm64.sh
+# Activate conda base environment for current session
+eval "$(/Users/gwpacworkstation/miniforge3/bin/conda shell.zsh hook)"
```

Comment:
Would it be better to change to `"$(/Users/<username>/miniforge3/bin/conda shell.zsh hook)"` ? I remember small things like this were confusing for me when I first started.

Reply:
Good catch! I meant to make such a change but missed it